routes/subjects.py

```
@@ -1,7 +1,7 @@
 from fastapi import APIRouter, Depends, HTTPException, status
 from models import Subject_db
 from middleWares import verify_device, admin_required
-from models import User, Widget_db, Component_db, Subject, Subject_db,Category_db
+from models import User, Widget_db, Component, Subject, Subject_db,Category_db
```

Ruff (0.8.2)

`models.User` imported but unused

Remove unused import: `models.User`

(F401)